src/hidpp20.h

```diff
@@ -619,6 +619,12 @@ enum hidpp20_color_led_zone_effect {
 #define HIDPP20_DPI_COUNT				5
 #define HIDPP20_LED_COUNT				2
 
+#define HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G402	0x01
+#define HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G303	0x02
+#define HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G900	0x03
+#define HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G915	0x04
+#define HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G705	0x05
```

Comment:
If you are gonna move this, might as well make this an enum along the way.

```suggestion
enum hidpp20_onboard_profiles_profile_type {
	HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G402 = 0x01,
	HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G303 = 0x02,
	HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G900 = 0x03,
	HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G915 = 0x04,
	HIDPP20_ONBOARD_PROFILES_PROFILE_TYPE_G705 = 0x05,
};
```